File: tracking-service-master/tracking-server/src/main/java/com/technicaltest/trackingserver/model/Stock.java
package com.technicaltest.trackingserver.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Stock {

    @JsonProperty("_id")
    private Long id;

    @JsonProperty("_quantity")
    private Integer quantity;

    @JsonProperty("_status")
    private String status;

    @JsonProperty("_idProduct")
    private Long idProduct;

    @JsonProperty("_idWarehouse")
    private Long idWarehouse;

    @JsonProperty("_nameWarehouse")
    private String nameWarehouse;

    @JsonProperty("_cityWarehouse")
    private String cityWarehouse;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Integer getQuantity() {
        return quantity;
    }

    public void setQuantity(Integer quantity) {
        this.quantity = quantity;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public Long getIdProduct() {
        return idProduct;
    }

    public void setIdProduct(Long idProduct) {
        this.idProduct = idProduct;
    }

    public Long getIdWarehouse() {
        return idWarehouse;
    }

    public void setIdWarehouse(Long idWarehouse) {
        this.idWarehouse = idWarehouse;
    }

    public String getNameWarehouse() {
        return nameWarehouse;
    }

    public void setNameWarehouse(String nameWarehouse) {
        this.nameWarehouse = nameWarehouse;
    }

    public String getCityWarehouse() {
        return cityWarehouse;
    }

    public void setCityWarehouse(String cityWarehouse) {
        this.cityWarehouse = cityWarehouse;
    }
}

File: tracking-service-master/tracking-web/src/app/model/stock.ts
export class Stock {
    _id: number;
    _quantity: number;
    _status: string;
    _idProduct: number;
    _idWarehouse: number;
    _nameWarehouse: string;
    _cityWarehouse: string;

    public get id(): number {
        return this._id;
    }

    public set id(id: number) {
        this._id = id;
    }

    public get quantity(): number {
        return this._quantity;
    }

    public set quantity(quantity: number) {
        this._quantity = quantity;
    }

    public get status(): string {
        return this._status;
    }

    public set status(status: string) {
        this._status = status;
    }

    public get idProduct(): number {
        return this._idProduct;
    }

    public set idProduct(idProduct: number) {
        this._idProduct = idProduct;
    }

    public get idWarehouse(): number {
        return this._idWarehouse;
    }

    public set idWarehouse(idWarehouse: number) {
        this._idWarehouse = idWarehouse;
    }

    public get nameWarehouse(): string {
        return this._nameWarehouse;
    }

    public set nameWarehouse(nameWarehouse: string) {
        this._nameWarehouse = nameWarehouse;
    }

    public get cityWarehouse(): string {
        return this._cityWarehouse;
    }

    public set cityWarehouse(cityWarehouse: string) {
        this._cityWarehouse = cityWarehouse;
    }
}